templates/gallery/gallery.amp.html

```diff
@@ -0,0 +1,434 @@
+<!doctype html>
+
+<!---
+Copyright 2017 The AMP Start Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+{{#gallery}}
+<html ⚡>
+  {{> ../../utils/amp-page-head.snip.html}}
+  <body>
+    {{> ./partials/header.snip.html}}
+    {{> ./partials/sidebar.snip.html}}
+    <main id="content" class="ampstart-gallery-base-content relative fit" role="main">
+      <h1 class="ampstart-gallery-landing-title flex justify-center items-center center absolute top-0 left-0 right-0 z1 mx-auto ampstart-gallery-base-invert">
+        {{{header.heading}}}
+      </h1>
+      <amp-carousel width="100vw" height="100vh" controls class="ampstart-gallery-landing-wall block relative">
+        <div class="ampstart-gallery-landing-wall-scroll relative">
+          <div class="ampstart-gallery-landing-image-wrap-0" >
+            <div class="ampstart-gallery-landing-image-transform">
+              <a href="./gallery-inner.amp.html" class="ampstart-gallery-landing-image-link"></a>
+              <amp-img
+                width="8"
+                height="12"
+                layout="responsive"
+                noloading
+                src="../../img/gallery/548241_200.jpg"
+                srcset="
+                  ../../img/gallery/548241_200.jpg 1x,
+                  ../../img/gallery/548241_400.jpg 2x
+                "
+              ></amp-img>
+            </div>
+            <div class="ampstart-gallery-landing-image-caption ampstart-gallery-landing-image-caption-bottom ampstart-gallery-landing-image-caption-extraleft">
```

Comment:
overall i think there is a lack of semantics here and everywhere else 

https://www.iandevlin.com/blog/2011/04/html5/html5-section-or-article

Please use semantics in HTML , plain divs are not good everywhere. 
http://html5doctor.com/lets-talk-about-semantics/